ivadomed/scripts/visualize_transforms.py

```diff
@@ -152,9 +154,9 @@ def run_visualization(input, config, number, output, roi):
                                               data_type="im")
 
             # Plot before / after transformation
-            fname_out = str(Path(output, stg_transforms+"slice"+str(i)+".png"))
-            print("Fname out: {}.".format(fname_out))
-            print("\t{}".format(dict(metadata)))
+            fname_out = str(Path(output, stg_transforms + "slice" + str(i) + ".png"))
+            print(f"Fname out: {fname_out}.")
+            print(f"\t{dict(metadata)}")
```

Comment:
Could you try to see which `logger info/debug` we can replace the `print` with here? I thought we replaced all print statements.